<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-IER6QUF" failures="1" tests="16" name="Test" time="70.734" errors="0" timestamp="2022-08-15T18:47:22 IST">
  <testcase classname="Test1.Test_D_3_Base_Test" name="@BeforeMethod setup" time="8.619">
    <failure type="org.openqa.selenium.NoSuchWindowException" message="no such window: target window already closed
from unknown error: web view not found
  (Session info: MicrosoftEdge=104.0.1293.54)
Build info: version: &amp;apos;4.3.0&amp;apos;, revision: &amp;apos;a4995e2c09*&amp;apos;
System info: host: &amp;apos;DESKTOP-IER6QUF&amp;apos;, ip: &amp;apos;192.168.43.81&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;18.0.1.1&amp;apos;
Driver info: org.openqa.selenium.edge.EdgeDriver
Command: [aac3f9f2c32714a8dd476df2b88b43dd, maximizeCurrentWindow {}]
Capabilities {acceptInsecureCerts: false, browserName: msedge, browserVersion: 104.0.1293.54, ms:edgeOptions: {debuggerAddress: localhost:50273}, msedge: {msedgedriverVersion: 104.0.1293.54 (cf00dc8720d8..., userDataDir: C:\Users\SMART\AppData\Loca...}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:50273/devtoo..., se:cdpVersion: 104.0.1293.54, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: aac3f9f2c32714a8dd476df2b88b43dd">
      <![CDATA[org.openqa.selenium.NoSuchWindowException: no such window: target window already closed
from unknown error: web view not found
  (Session info: MicrosoftEdge=104.0.1293.54)
Build info: version: '4.3.0', revision: 'a4995e2c09*'
System info: host: 'DESKTOP-IER6QUF', ip: '192.168.43.81', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '18.0.1.1'
Driver info: org.openqa.selenium.edge.EdgeDriver
Command: [aac3f9f2c32714a8dd476df2b88b43dd, maximizeCurrentWindow {}]
Capabilities {acceptInsecureCerts: false, browserName: msedge, browserVersion: 104.0.1293.54, ms:edgeOptions: {debuggerAddress: localhost:50273}, msedge: {msedgedriverVersion: 104.0.1293.54 (cf00dc8720d8..., userDataDir: C:\Users\SMART\AppData\Loca...}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:50273/devtoo..., se:cdpVersion: 104.0.1293.54, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: aac3f9f2c32714a8dd476df2b88b43dd
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:569)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:624)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:628)
at org.openqa.selenium.remote.RemoteWebDriver$RemoteWebDriverOptions$RemoteWindow.maximize(RemoteWebDriver.java:1027)
at Test1.Test_D_3_Base_Test.setup(Test_D_3_Base_Test.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod setup -->
  <testcase classname="Test1.Test_D_3_Base_Test" name="@AfterMethod tearDown" time="-1.660569442353E9">
    <skipped/>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase classname="Test1.Test_D_6_TestNG_Suite" name="T1" time="0.018"/>
  <testcase classname="Test1.Test_D_6_TestNG_Suite" name="T2" time="0.004"/>
  <testcase classname="Test1.Test_D_6_TestNG_Suite" name="T3" time="0.003"/>
  <testcase classname="Test1.Test_D_2" name="Test1" time="0.007"/>
  <testcase classname="Test1.Test_D_2" name="Test2" time="0.004"/>
  <testcase classname="Test1.Test_D_4_Child_class" name="p1" time="0.003"/>
  <testcase classname="Test1.Test_D_4_Child_class" name="p2" time="0.011"/>
  <testcase classname="Test1.Test_D_4_Child_class" name="p3" time="0.002"/>
  <testcase classname="Test1.Test_D_4_Child_class" name="p4" time="0.0">
    <skipped/>
  </testcase> <!-- p4 -->
  <testcase classname="Test1.Test_D_5_Keywords" name="C" time="0.113">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Test1.Test_D_5_Keywords.C(Test_D_5_Keywords.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- C -->
  <testcase classname="Test1.Test_D_5_Keywords" name="D" time="0.0">
    <skipped/>
  </testcase> <!-- D -->
  <testcase classname="Test1.Test_D_5_Keywords" name="E" time="0.002"/>
  <testcase classname="Test1.Test_D_5_Keywords" name="E" time="0.002"/>
  <testcase classname="Test1.Test_D_5_Keywords" name="E" time="0.002"/>
  <testcase classname="Test1.Test_D_5_Keywords" name="E" time="0.004"/>
  <testcase classname="Test1.Test_D_5_Keywords" name="A" time="0.007"/>
  <testcase classname="Test1.Test_D_5_Keywords" name="B" time="0.0">
    <ignored/>
  </testcase> <!-- B -->
</testsuite> <!-- Test -->
